tests/components/kodi/__init__.py

```diff
@@ -1,43 +1,15 @@
 """Tests for the Kodi integration."""
 from unittest.mock import patch
 
-from homeassistant.components.kodi.const import CONF_WS_PORT, DOMAIN
-from homeassistant.const import (
-    CONF_HOST,
-    CONF_NAME,
-    CONF_PASSWORD,
-    CONF_PORT,
-    CONF_SSL,
-    CONF_USERNAME,
-)
+from .conftest import PATCH_KODI_CONNMAN
```

Comment:
`conftest` is imported by default, so no need to import explicitly

Reply:
It does not affect `__init__.py` (if there are no `test_` functions inside, I think). If I remove the import:
```
E       NameError: name 'PATCH_KODI_CONNMAN' is not defined

__init__.py:13: NameError
```

Reply:
OK. Ideally `init_integration` would be a fixture in itself and move to `conftest.py` but that's probably beyond the scope for this PR.

Reply:
Acutally `init_integration` has become a [fixture](https://github.com/home-assistant/core/blob/e8d3a786c38c3bd3e105c13162f1f34ed1fd5754/tests/components/kodi/conftest.py#L80) and interacts with these custom markers.

Reply:
Then you can move it to `conftest.py` and then you don't have to import `PATCH_KODI_CONNMAN`